pkg/resources/task.go

```diff
@@ -619,6 +635,21 @@ func UpdateTask(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if d.HasChange(AllowOverlappingExecution) {
+		var q string
+		n := d.Get(AllowOverlappingExecution)
```

Comment:
why not use d.GetChange() instead? https://pkg.go.dev/github.com/hashicorp/terraform/helper/schema#ResourceData.GetChange.